src/main/java/com/urswolfer/gerrit/client/rest/http/changes/ChangeApiRestClient.java

```diff
@@ -17,19 +17,16 @@
 package com.urswolfer.gerrit.client.rest.http.changes;
 
 import com.google.common.collect.Iterables;
-import com.google.gerrit.extensions.api.changes.AbandonInput;
-import com.google.gerrit.extensions.api.changes.AddReviewerInput;
-import com.google.gerrit.extensions.api.changes.ChangeApi;
-import com.google.gerrit.extensions.api.changes.FixInput;
-import com.google.gerrit.extensions.api.changes.RestoreInput;
-import com.google.gerrit.extensions.api.changes.RevisionApi;
+import com.google.gerrit.extensions.api.changes.*;
```

Comment:
Please try to keep imports as much the same as possible in order to keep the diff small.